extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/LDAPGuacamoleProperties.java

```diff
@@ -145,4 +145,16 @@ private LDAPGuacamoleProperties() {}
 
     };
 
+    /**
+     * The maximum number of results a LDAP query can return.
+     */
+    public static final IntegerGuacamoleProperty LDAP_MAX_SEARCH_RESULTS = new IntegerGuacamoleProperty() {
+
+        @Override
+        public String getName() { return "ldap-maxresults"; }
```

Comment:
Just noticed:

`LDAP_MAX_SEARCH_RESULTS` is very nice and readable, so why is the property just `ldap-maxresults`? (Nice dash following the "ldap", but not between "max" and "results"). For consistency with other properties used by Guacamole, this should at least be `ldap-max-results`, though given the nice `LDAP_MAX_SEARCH_RESULTS` name, I'd suggest `ldap-max-search-results` instead.